sdcm/cluster.py

```diff
@@ -3490,6 +3494,7 @@ def node_setup(self, node, verbose=False, timeout=3600, wait_db_up=True):  # pyl
             self._scylla_post_install(node, install_scylla)
 
             node.stop_scylla_server(verify_down=False)
+            node.remoter.run("sudo rm -rf /var/lib/scylla/data/*")  # Clear data folder to drop wrong cluster name data.
```

Comment:
I think @aleksbykov recently removed it, we should put this under some condition, maybe by cluster_backend...

Reply:
No, this line presents in Docker's `node_setup()`, but not here.  It's pretty safe to have it in base version: it's OK to wipe data from previous runs because we want to have fresh Scylla anyway.